GG/src/Font.cpp

```diff
@@ -1167,47 +1167,28 @@ std::string Font::StripTags(const std::string& text, bool strip_unpaired_tags)
 {
     using namespace boost::xpressive;
 
-    bool temp_bool = false;
-    std::stack<Substring> tag_stack;
-    MatchesKnownTag matches_known_tag(StaticTagHandler().KnownTags(), temp_bool);
-    MatchesTopOfStack matches_tag_stack(tag_stack, temp_bool);
+    sregex & regex = StaticTagHandler().Regex(text, false, strip_unpaired_tags);
 
     mark_tag tag_name_tag(1);
```

Comment:
Not for this PR, but shouldn't those be part of TagHandler? There are at lease two other places where this is duplicated.